internal/painter/gl/draw.go

```diff
@@ -278,6 +376,40 @@ func rectInnerCoords(size fyne.Size, pos fyne.Position, fill canvas.ImageFill, a
 	return size, pos
 }
 
+func (p *painter) vecRectCoords(
+	pos fyne.Position,
+	rect *canvas.Rectangle,
+	frame fyne.Size,
+) []float32 {
```

Comment:
Same as the comment above.

Reply:
I changed the formating of importing parameters to one line, but IMO the readability of the func is much better if you use more lines.  
Example one line:
```Go
func (p *painter) lineCoords(pos, pos1, pos2 fyne.Position, lineWidth, feather float32, frame fyne.Size) ([]float32, float32, float32) {
	// Shift line coordinates so that they match the target position.
	xPosDiff := pos.X - fyne.Min(pos1.X, pos2.X)
```
The line seems not to have an end and the func signuture is confusing.